configserver/src/main/java/com/yahoo/vespa/config/server/ApplicationRepository.java

```diff
@@ -218,6 +218,7 @@ private ApplicationRepository(TenantRepository tenantRepository,
         private Metric metric = new NullMetric();
         private SecretStoreValidator secretStoreValidator = new SecretStoreValidator(new SecretStoreProvider().get());
         private FlagSource flagSource = new InMemoryFlagSource();
+        private ConfigConvergenceChecker configConvergenceChecker = new ConfigConvergenceChecker();
```

Comment:
This component is injectable, probably because it holds an executor. Perhaps use this?

On a side note, that executor isn't closed (no `deconstruct()` method), which should also be fixed.

Reply:
Will look into it